crates/noirc_evaluator/src/lib.rs

```diff
@@ -169,6 +169,10 @@ impl<'a> Evaluator<'a> {
                 let err = RuntimeErrorKind::Unimplemented("The Or operation is currently not implemented. First implement in Barretenberg.".to_owned());
                 Err(err)
             }
+            HirBinaryOpKind::Shr | HirBinaryOpKind::Shl => {
+                let err = RuntimeErrorKind::Unimplemented("Bit shift operations are not currently implemented.".to_owned());
+                Err(err)
```

Comment:
nitpick but lets get rid of the `err` variable here, it's name is unhelpful and it'd be shorter to just wrap the RuntimeErrorKind in Err.

Reply:
Done